src/app/modules/layout/topbar/directory-services-indicator/directory-services-indicator.component.ts

```diff
@@ -72,12 +75,20 @@ export class DirectoryServicesIndicatorComponent implements OnInit, OnDestroy {
 
   private loadDirectoryServicesStatus(): void {
     // TODO: Sync endpoints
```

Comment:
Looks good!
One more thing, since these changes already affect related code, I suggest to sync 2 calls in this method as suggested by a `TODO`, if this is possible to do within this PR

Reply:
Middleware still sent events on `directoryservices.status` endpoint for changes.

https://github.com/truenas/middleware/blob/f5677a887ee6529a13d2b5e8c44abe1af7d753b3/src/middlewared/middlewared/plugins/directoryservices.py#L242-L248